src/modules/builderView/configuration/AddImageStep.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+
+import {
+  Box,
+  Button,
+  InputLabel,
+  Stack,
+  TextField,
+  Typography,
+} from '@mui/material';
+
+import { NameTheFrameSettingsNames } from '@/@types';
+import { useNameFrameTranslation } from '@/config/i18n';
+import { hooks, mutations } from '@/config/queryClient';
+import { DESCRIPTION_INPUT_ID } from '@/config/selectors';
+import { NAME_THE_FRAME } from '@/langs/constants';
+import ImageDisplay from '@/modules/common/ImageDisplay';
+import UploadImage from '@/modules/common/UploadImage';
+
+type Props = {
+  moveToNextStep: () => void;
+};
+const AddImageStep = ({ moveToNextStep }: Props): JSX.Element => {
+  const { t } = useNameFrameTranslation();
```

Comment:
To avoid name dependent naming.
```suggestion
  const { t } = useAppTranslation();
```